lib/pharos/config_schema.rb

```diff
@@ -59,9 +59,19 @@ def self.messages
               optional(:ssh_proxy_command).filled
               optional(:container_runtime).filled(included_in?: ['docker', 'custom_docker', 'cri-o'])
               optional(:environment).filled
+              optional(:bastion).schema do
+                required(:address).filled(:str?)
+                optional(:user).filled(:str?)
+                optional(:ssh_key_path).filled(:str?)
+              end
             end
           end
         end
+        optional(:bastion).schema do
```

Comment:
We don't have this sort of "`hosts`-global" settings for any other host feature.

We could make a generic one at some point.

Reply:
@jnummelin WDYT?

Reply:
Would be nice to have global host settings. So maybe move `bastion` into something like:
```
host_config:
  bastion: ...
```

No need to support all the options in this PR, we can later add other PRs to support all host options on global level

Reply:
Ok, I'm going to remove "global" bastion then from this pr.. let's open a separate issue about `host_config:`.